tests/main/base-invalid-type/task.yaml

```diff
@@ -1,7 +1,7 @@
 summary: Check that installing a snap requiring a base fails if base has invalid type
 
 prepare: |
-    snap pack "$TESTSLIB/snaps/test-snapd-invalid-base"
+    snap pack "$PWD"/test-snapd-invalid-base
```

Comment:
$PWD is unneeded, is it? Also the other tests below don't seem to be using it consistently.

Reply:
you are right, it is not needed, I'll remove it, thanks!